src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -98,7 +98,7 @@ constexpr size_t minimum_number_of_points =
  * \brief Maximum number of allowed collocation points.
  */
 template <Basis>
-constexpr size_t maximum_number_of_points = 12;
+constexpr size_t maximum_number_of_points = 16;
```

Comment:
is this necessary now?  We thought your tests showed 12 points was more than sufficient...

Reply:
some of the tests seemed to 'just' reach convergence at 12 -- that is, they showed improvement from 8 to 10, but little improvement from 10 to 12. To be reasonably flexible to future CCE studies, it seemed best to allow up to 16. 
If you both feel that is still over-reaching, I can drop this commit and leave the spectral density discussion for the future.

Reply:
let's leave it at 12 for now...